vllm/core/evictor_v2.py

```diff
@@ -1,6 +1,9 @@
 import enum
+import heapq
 from abc import ABC, abstractmethod
-from typing import OrderedDict, Tuple
+from typing import Dict, List, Tuple
+
+CLEANUP_THRESHOLD = 50
```

Comment:
I would make this a static class member, since it is used only inside the scope of the class below.

Reply:
Thank you, I fixed this